### ISSUE TITLE
Uncart not dumping private header

### ISSUE THREAD

**Issue by Dimensional** [citra-emu/uncart#22]
The program doesn't dump the private headers of my games with the game itself, which is shown when I run the dump through a header reader. Given these dumps will be solely used privately, the headers would be completely unique, and the risk of a ban with any online feature would be negligible. Can the program be updated to dump the private headers with the game? Or even put it into a text file for use with carts like Sky3DS(Plus)?

**Comment by yuriks** [comment]
As far as I know the "private header" isn't part of the actual cart image, and I don't know how to dump it if it exists. (I briefly looked into this when initially fixing the dumping.)

**Comment by Dimensional** [comment]
Every game cart has a unique private header. It's why people have been getting banned when using things like Sky3DS or Gateway to play rom dumps. All because everybody was using the same dump with the same headers.

**Comment by yuriks** [comment]
If the "private header" is the few bytes of data stored after the NCSD header, @Normmatt told me that's just some stuff added by the Gateway dumper, it's not part of the ROM.

**Comment by shinyquagsire23** [comment]
@yuriks I'm pretty sure it's also unused ROM, regardless, and Gateway did set a bit of a standard in terms of placement (for better or for worse). Worst case scenario, dump it to a separate file? Wouldn't be very neat though and people would just be grafting it into their dumps anyhow.

**Comment by yuriks** [comment]
@shinyquagsire23 Right, but my point is that it's not part of the ROM, and *I* don't know how to dump it or where it comes from either.
